resources/metadata/whitesource.yaml

```diff
@@ -0,0 +1,545 @@
+      default: null
+    - name: userToken
+      type: string
+      description: WhiteSource token identifying the user executing the scan
+      mandatory: true
+      scope:
+        - GENERAL
+        - PARAMETERS
+        - STAGES
+        - STEPS
+      default: null
+    - name: licensingVulnerabilities
+      type: bool
+      description: Whether license compliance is considered and reported as part of the assessment.
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: true
+    - name: agentFileName
+      type: string
+      description: Locally used name for the Unified Agent jar file after download.
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: wss-unified-agent.jar
+    - name: emailAddressesOfInitialProductAdmins
+      type: string
+      description: The list of email addresses to assign as product admins for newly created WhiteSource products.
+      mandatory: false
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: []
+    - name: productVersion
+      type: string
+      description: Version of the WhiteSource product to be created and used for results aggregation, usually determined automatically.
+      mandatory: false
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: null
+    - name: jreDownloadUrl
+      type: string
+      description: URL used for downloading the Java Runtime Environment (JRE) required to run the WhiteSource Unified Agent.
+      mandatory: false
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: null
+    - name: productName
+      type: string
+      description: Name of the WhiteSource product to be created and used for results aggregation.
+      mandatory: true
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: null
+    - name: verbose
+      type: bool
+      description: Whether verbose output should be produced.
+      mandatory: false
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: false
+    - name: projectName
+      aliases:
+        - name: whitesourceProjectName
+      type: string
+      description: "The project used for reporting results in Whitesource"
+      scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+      default: '{{list .GroupID .ArtifactID | join "-" | trimAll "-"}}'
+    - name: projectToken
+      type: string
+      description: Project token to execute scan on
+      mandatory: false
+      scope:
+        - GENERAL
+        - PARAMETERS
+        - STAGES
+        - STEPS
+      default: null
+    - name: projectName
+      type: string
+      description: Project name to execute scan on
+      mandatory: false
+      scope:
+        - GENERAL
+        - PARAMETERS
+        - STAGES
+        - STEPS
+      default: null
```

Comment:
Why is this specified another time?

Reply:
Good catch, probably a copy and paste error on my behalf.